### PR TITLE
990 Fix warning level Codacy found issues: Warn when a heading ends with a group of characters.

### PR DESCRIPTION
**Pull request by tsvetaevAlex**
https://github.com/jdi-testing/jdi-light/issues/990

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.epam.jdi</groupId>
-    <version>1.1.0</version>
+    <version>1.1.5</version>
```
Comment (AUlasen) [inline pom.xml]: ???

File: pom.xml [hunk 2]
```diff
@@ -49,16 +54,15 @@
...
         <module>jdi-light-html-tests</module>
         <module>jdi-light-html</module>
-        <module>jdi-light-bootstrap</module>
-        <module>jdi-light-bootstrap-tests</module>
-        <module>jdi-light-bdd</module>
-        <module>jdi-light-bdd-tests</module>
-        <module>jdi-light-bdd-no-po-tests</module>
+        <module>jdi-bdd</module>
+        <module>jdi-bdd-tests</module>
+        <module>jdi-bdd-no-po-tests</module>
```
Comment (AUlasen) [inline pom.xml]: Why are you editing pom file?

File: README.md [hunk 3]
```diff
@@ -41,31 +41,33 @@ Run test execution automatically and correct tests if any changes in layout happ
...
-Facebook: <a href = "https://www.facebook.com/groups/jdi.framework/" target = "a_blank"> https://www.facebook.com/groups/jdi.framework/ </a>
+Facebook: https://www.facebook.com/groups/jdi.framework/
 
 You can ask your questions on StackOverflow with [![htmlelements](https://img.shields.io/badge/stackoverflow-jdiframework-orange.svg?style=flat)](http://stackoverflow.com/questions/tagged/jdiframework) tag
 
-Get Help in active users Skype chat: <a href = "https://join.skype.com/clvyVvnZvWqc" target = "a_blank"> https://join.skype.com/clvyVvnZvWqc </a>
+Get Help in active users Skype chat: https://join.skype.com/clvyVvnZvWqc
 
-## Contacts:
+## Contacts :
```
Comment (AUlasen) [inline README.md]: It cant solve the problem, trailing `:` was not disappeared.

File: README.md [hunk 4]
```diff
@@ -10,12 +10,12 @@
...
-Tutorial repo: <a href = "https://github.com/jdi-tutorials" target = "a_blank"> https://github.com/jdi-tutorials </a> </br>
-Project templates: <a href = "https://github.com/jdi-templates" target = "a_blank"> https://github.com/jdi-templates </a>  - for fast projects start </br>
-Examples: <a href = "https://github.com/jdi-examples" target = "a_blank"> https://github.com/jdi-examples </a>  </br> 
+Documentation: https://jdi-docs.github.io/jdi-light/?java - different types of documentation from introduction and tutorial to technical methods documentation </br>
+Introduction + fast start example: https://jdi-docs.github.io/jdi-light/?java#introduction </br>
+Tutorial: https://jdi-docs.github.io/jdi-light/?java#tutorial - helps to get main ideas on practice </br>
+Tutorial repo:  https://github.com/jdi-tutorials </br>
+Project templates: https://github.com/jdi-templates - for fast projects start </br>
+Examples: https://github.com/jdi-examples </br>
 How to use [UI Elements examples](https://github.com/jdi-testing/jdi-light/tree/master/jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements) </br>
```
Comment (AUlasen) [inline README.md]: It doesn't connect to issue 990. And it causes a lot of new Codacy issues:
https://app.codacy.com/manual/jdi-testing/jdi-light/pullRequest?prid=4361169

File: README.md [hunk 5]
```diff
@@ -26,12 +26,12 @@ Increase test [performance examples](https://github.com/jdi-testing/jdi-light/tr
...
+- Has simple integration with any Selenium based project that allows to increase projects stability, add user actions logs and enrich standard WebElements capabilities with new features in minutes (TBD add Guide)
+- Extends Page Objects pattern with powerful UI Elements: Buttons, TextFields, Forms, Tables etc ( TBD add good links to examples)
+- Write stable and predictable tests that fail only because of changing business logic or layout. No more waits, thread sleep and other flaky staff
+- Increase your tests execution speed in general and especially for complex cases like search in big tables, dropdowns with hundreds elements or enter text with thousand lines
+- Get detailed logs and good reports of all user actions with no effort
+- Easy integration with all modern Automation tools: CI (Jenkins, TC etc.), Logging (Log4j or any other slf4j based logs), Reporting (Allure or Report Portal); Browser/Devices farms (Selenium Grid, Browser Stack, Selenoid etc.), Test Runners (TestNG, JUnit) etc.
```
Comment (AUlasen) [inline README.md]: It doesn't connect to issue 990. And it causes a lot of new Codacy issues:
https://app.codacy.com/manual/jdi-testing/jdi-light/pullRequest?prid=4361169

File: README.md [hunk 6]
```diff
@@ -41,31 +41,33 @@ Run test execution automatically and correct tests if any changes in layout happ
...
-VK: <a href = "https://vk.com/jdi_framework" target = "a_blank"> https://vk.com/jdi_framework </a>
+VK: https://vk.com/jdi_framework
 
-Facebook: <a href = "https://www.facebook.com/groups/jdi.framework/" target = "a_blank"> https://www.facebook.com/groups/jdi.framework/ </a>
+Facebook: https://www.facebook.com/groups/jdi.framework/
 
 You can ask your questions on StackOverflow with [![htmlelements](https://img.shields.io/badge/stackoverflow-jdiframework-orange.svg?style=flat)](http://stackoverflow.com/questions/tagged/jdiframework) tag
 
-Get Help in active users Skype chat: <a href = "https://join.skype.com/clvyVvnZvWqc" target = "a_blank"> https://join.skype.com/clvyVvnZvWqc </a>
+Get Help in active users Skype chat: https://join.skype.com/clvyVvnZvWqc
```
Comment (AUlasen) [inline README.md]: It doesn't connect to issue 990. And it causes a lot of new Codacy issues:
https://app.codacy.com/manual/jdi-testing/jdi-light/pullRequest?prid=4361169

File: pom.xml [hunk 7]
```diff
@@ -22,6 +22,11 @@
             <artifactId>coveralls-maven-plugin</artifactId>
             <version>4.3.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.github.demidenko05</groupId>
+            <artifactId>a-javabeans</artifactId>
+            <version>1.0.4</version>
+        </dependency>
```
Comment (AUlasen) [inline pom.xml]: ???

File: pom.xml (hunk 2 above)
Comment (tsvetaevAlex) [reply pom.xml]: import sun.reflect.generics.reflectiveObjects.NotImplementedException;
Error:(14, 46) java: package sun.reflect.generics.reflectiveObjects does not exist

File: pom.xml (hunk 7 above)
Comment (tsvetaevAlex) [reply pom.xml]: это было еще до того, как меня подключили к этому проекту

File: README.md (hunk 3 above)
Comment (tsvetaevAlex) [reply README.md]: However warning disappeared

Comment (tsvetaevAlex) [reply README.md]: image

**Comment by AlexeyGirin** [comment]
Not actual any more

**Pull request closed without merging**